lib/rateitapp.rb

```diff
@@ -4,4 +4,5 @@
 
 # Engine namespace
 module Rateitapp
+  mattr_accessor :oauth_domain
```

Comment:
This should probably be called `provider_domain`, since technically it is the domain of the oauth provider we are passing.